packages/server/tests/acceptance.spec.ts

```diff
@@ -360,14 +372,50 @@ describe('RPC Server Integration Tests', async function () {
         expect(res.data.result).to.be.equal(false);
     });
 
-    it('should execute eth_sendRawTransaction legacy', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', ['0x' + legacyTransactionHex]);
-        expect(res.data.result).to.be.equal('0x9ffbd69c44cf643ed8d1e756b505e545e3b5dd3a6b5ef9da1d8eca6679706594');
-    });
-
-    it('should execute "eth_sendRawTransaction" london', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', ['0x' + londonTransactionHex]);
-        expect(res.data.result).to.be.equal('0xcdbbfb6400aab319f97d32c38e285f0d0399c2b48b683f04878b5f07eb0d50e3');
+    it('should execute "eth_sendRawTransaction" for legacy transactions', async function () {
+        const signedTx = await utils.signRawTransaction({
+            to: mirrorContract.evm_address,
+            value: 1,
+            chainId: 0x12a,
+            gasPrice: 720000000000,
+            gasLimit: 3000000
+        }, ethCompPrivateKey3);
+
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', [signedTx]);
+        expect(res.data.result).to.be.equal('0x93c4b87f7fe3d6071a9c58acf5b64ec976c60ca2017f21fac42f445472885727');
+    });
+
+    it('should fail "eth_sendRawTransaction" for eip155 transactions', async function () {
+        // INVALID_ETHEREUM_TX
+        const signedTx = await utils.signRawTransaction({
+            to: mirrorContract.evm_address,
+            value: 1,
+            chainId: 0x12a,
+            gasPrice: 720000000000,
+            gasLimit: 3000000,
+            nonce: 1,
+            type: 1
+        }, ethCompPrivateKey3);
+
+        const res = await utils.callFailingRelayMethod(this.relayClient, 'eth_sendRawTransaction', [signedTx]);
+        expect(res.data.error.message).to.be.equal('Internal error');
+        expect(res.data.error.code).to.be.equal(-32603);
+    });
+
+    it('should execute "eth_sendRawTransaction" for London transactions', async function () {
+        const signedTx = await utils.signRawTransaction({
+            to: mirrorContract.evm_address,
```

Comment:
nit: move object out to `defaultLondonTransactionData`. Then you override `to` here